ansible_collections/arista/avd/examples/l2ls-fabric/README.md

```diff
@@ -0,0 +1,137 @@
+# Example for L2LS Fabric
+
+## Introduction
+
+This example is meant to explore using AVD with an L2LS (Layer 2 Leaf Spine) Topology. The example topology shown below contains 2 Spines and 2 Leaf Pairs. The L2LS Fabric is purely Layer 2 with no routing of SVIs in the Fabric.  Later we will add in L3 routing functionality to this example`.
```

Comment:
```suggestion
This example is meant to explore using AVD with an L2LS (Layer 2 Leaf Spine) Topology. The example topology shown below contains 2 Spines and 2 Leaf Pairs. The L2LS Fabric is Layer 2 only with no routing of SVIs in the Fabric.  Later L3 routing functionality will be added to this example.
```

Reply:
Hey @gmuloc. I am still in draft mode of writing the docs here.  Appreciate the comments.  I was rewriting much of this  today.   Please take a look now and see how it reads.  I plan to mark this PR ready for review next week.